modules/tensor_mechanics/tests/static_deformations/tests

```diff
@@ -11,4 +11,12 @@
     exodiff = 'cosserat_tension_out.e'
     abs_zero = 1e-9
   [../]
+
+  [./except01]
+    type = RunException
+    input = cosserat_tension.i
+    cli_args = 'Kernels/x_moment/component=1 Outputs/file_base=except01'
+    expect_err = 'MomentBalancing: The variable for this Kernel must be equal to the Cosserat rotation variable defined by the "component" and the "Cosserat_rotations" parameters'
+    prereq = tension
+  [../]
 []
```

Comment:
Super nit-picky but would you mind adding a blank line here? We need to start scanning these "tests" files for this problem like we do everything else.